db/migrate/20160418141210_add_read_only_to_miq_alert.rb

```diff
@@ -1,9 +1,17 @@
+require 'miq_expression.rb'
+
 class AddReadOnlyToMiqAlert < ActiveRecord::Migration[5.0]
+  class MiqAlert < ActiveRecord::Base; end
+
+  def self.guids_from_yaml
+    YAML.load_file("db/fixtures/miq_alerts.yml").map { |h| h["guid"] || h[:guid] }
```

Comment:
>   we should add read_only to the yml

I think the seed code should hard-set read_only and not rely on it being in the yml or not.